tooling/minimize/src/ty.rs

```diff
@@ -1,5 +1,11 @@
+use std::cell::RefCell;
+
 use crate::*;
 
+thread_local! {
+    static TY_CACHE: RefCell<HashMap<smir::Ty, Type>> = RefCell::new(HashMap::new())
+}
```

Comment:
The cache could live in `Ctxt`, no need to make it a thread-local

Reply:
moved it to `Ctxt`